### PR TITLE
Job Aids for Windows Server 2003 Deployment Kit

### LINKED ISSUE #3592: 404 on Creating a Site Design
There's a 404 for the link the Job Aids: https://go.microsoft.com/fwlink/?LinkID=102558

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 6c7feb50-baf0-4041-25f5-8fa96029d418
* Version Independent ID: 9426a72a-e27f-aadd-e01e-b73d3849f22f
* Content: [Creating a Site Design](https://docs.microsoft.com/en-us/windows-server/identity/ad-ds/plan/creating-a-site-design#feedback)
* Content Source: [WindowsServerDocs/identity/ad-ds/plan/Creating-a-Site-Design.md](https://github.com/MicrosoftDocs/windowsserverdocs/blob/master/WindowsServerDocs/identity/ad-ds/plan/Creating-a-Site-Design.md)
* Product: **windows-server**
* Technology: **identity-adds**
* GitHub Login: @MicrosoftGuyJFlo
* Microsoft Alias: **joflore**

### PR DESCRIPTION
**Pull request by illfated**
**Description:**

As noted in PR #4167 (**Selecting the Forest Root Domain: updated links**), I have been informed as follows: "We typically are trying to replace the fwlinks with direct URLs as we go through files. If you happen to see them and feel like it, you can just replace the whole fwlink with the direct URL."

This PR aims to correct all occurrences of the specific link to the download page for "Job Aids for Windows Server 2003 Deployment Kit".

Thanks to @eross-msft (Elizabeth Ross) for the valuable feedback.

I have noticed that many other links are either outdated or broken, but I have not yet had time to extend my research to those other links yet. If you know the correct new link address for any of the outdated links in any file in this Pull Request, feel free to tell me about it, so I may add that/those link correction(s) as well.

**Changes proposed:**
- Replace https://go.microsoft.com/fwlink/?LinkID=102558 with https://www.microsoft.com/download/details.aspx?id=9608 anywhere
- Change the page name reference text to link text for existing links, instead of the link being used both as link text and the link itself.
- Remove any redundant end-of-line (EOL) whitespace
- Add MarkDown indent marker compatibility spacing (1 space) anywhere
- Standardize bullet point list internal indent spacing from 3 to 1

**Ticket closure and reference:**

Closes #3592 #3760 #3964
Ref.: #4147, #4167

### CODE REVIEW

**Comment by illfated** [comment]
Looks like I didn't manage to write the closing command correctly in my commit message. I might try to improve it, but in case I can't resolve how the command to close multiple tickets with 1 Close command, we may just need to close the tickets #3760 and #3964 manually.

**Comment by illfated** [comment]
@JohanFreelancer9 : I keep forgetting whether or not this repository belongs to your workspace, but if you would be so kind to perform a copy review to look for anything simple typos or naming conventions I may have missed out on, I would be grateful for your contribution. If you are aware of any link improvements to be included as well, that would be a nice bonus.

**Comment by illfated** [comment]
Additional change, not mentioned in my [**Changes proposed**] list: in row 34 of the PR file [`identity/ad-ds/plan/Finding-Additional-Resources-for-Logical-Structure-Design.md`](https://github.com/MicrosoftDocs/windowsserverdocs/pull/4181/files?diff=unified&w=1#diff-078ade3b36a48e88964afbbf2fe0f024), I chose to split that row into 2 separate lines, to keep the page format consistent by having only 1 topic and 1 linked text for each bullet point in the list:
- https://github.com/MicrosoftDocs/windowsserverdocs/pull/4181/files?diff=unified&w=1#diff-078ade3b36a48e88964afbbf2fe0f024

File: WindowsServerDocs/identity/ad-ds/plan/Assigning-Domain-Names.md [hunk 1]
```diff
@@ -12,26 +12,23 @@ ms.technology: identity-adds
...
-  
-Document the DNS prefix and NetBIOS names that you select for each domain in your forest. You can add the DNS and NetBIOS name information to the "Domain Planning" worksheet that you created to document your plan for new and upgraded domains. To open it, download Job_Aids_Designing_and_Deploying_Directory_and_Security_Services.zip from Job Aids for Windows Server 2003 Deployment Kit ([https://go.microsoft.com/fwlink/?LinkID=102558](https://go.microsoft.com/fwlink/?LinkID=102558)) and open "Domain Planning" (DSSLOGI_5.doc).  
-  
+> Applies To: Windows Server 2016, Windows Server 2012 R2, Windows Server 2012
 
+You must assign a name to every domain in your plan. Active Directory Domain Services (AD DS) domains have two types of names: Domain Name System (DNS) names and NetBIOS names. In general, both names are visible to end users. The DNS names of Active Directory domains include two parts, a prefix and a suffix. When creating domain names, first determine the DNS prefix. This is the first label in the DNS name of the domain. The suffix is determined when you select the name of the forest root domain. The following table lists the prefix naming rules for DNS names.
```
Comment (JohanFreelancer9) [inline WindowsServerDocs/identity/ad-ds/plan/Assigning-Domain-Names.md:17]:
```suggestion
You must assign a name to every domain in your plan. Active Directory Domain Services (AD DS) domains have two types of names: Domain Name System (DNS) names and NetBIOS names. In general, both names are visible to end-users. The DNS names of Active Directory domains include two parts, a prefix and a suffix. When creating domain names, first determine the DNS prefix. This is the first label in the DNS name of the domain. The suffix is determined when you select the name of the forest root domain. The following table lists the prefix naming rules for DNS names.
```

Comment (JohanFreelancer9) [reply WindowsServerDocs/identity/ad-ds/plan/Assigning-Domain-Names.md:17]: Change "end users" to "end-users".

File: WindowsServerDocs/identity/ad-ds/plan/Assigning-Domain-Names.md [hunk 2]
```diff
@@ -12,26 +12,23 @@ ms.technology: identity-adds
...
+|Rule|Explanation|
+|--------|---------------|
+|Select a prefix that is not likely to become outdated.|Avoid names such as a product line or operating system that might change in the future. We recommend using geographical names.|
+|Select a prefix that includes Internet standard characters only.|A-Z, a-z, 0-9, and (-), but not entirely numerical.|
+|Include 15 characters or less in the prefix.|If you choose a prefix length of 15 characters or less, the NetBIOS name is the same as the prefix.|
+
+For more information, see [Naming conventions in Active Directory for computers, domains, sites, and OUs](https://go.microsoft.com/fwlink/?LinkId=106629).
+
+> [!NOTE]
+> Although Dcpromo.exe in Windows Server 2008 and Windows Server 2003 allows you to create a single-label DNS domain name, you should not use a single-label DNS name for a domain for several reasons. In Windows Server 2008 R2, Dcpromo.exe does not allow you to create a single-label DNS name for a domain. For more information, see [https://go.microsoft.com/fwlink/?LinkId=92467.](https://go.microsoft.com/fwlink/?LinkId=92467)
```
Comment (JohanFreelancer9) [inline WindowsServerDocs/identity/ad-ds/plan/Assigning-Domain-Names.md:28]:
```suggestion
> Although Dcpromo.exe in Windows Server 2008 and Windows Server 2003 allows you to create a single-label DNS domain name, you should not use a single-label DNS name for a domain for several reasons. In Windows Server 2008 R2, Dcpromo.exe does not allow you to create a single-label DNS name for a domain. For more information, see [Deployment and operation of Active Directory domains that are configured by using single-label DNS names](https://go.microsoft.com/fwlink/?LinkId=92467).
```

File: WindowsServerDocs/identity/ad-ds/plan/Assigning-Domain-Names.md (hunk 1 above)
Comment (illfated) [reply WindowsServerDocs/identity/ad-ds/plan/Assigning-Domain-Names.md:17]: Hmm, thanks. I have never used "end-user" specifically myself, so I had to look it up: https://en.wikipedia.org/wiki/End_user
https://github.com/MicrosoftDocs/windowsserverdocs/search?q=%22end+user%22&unscoped_q=%22end+user%22

To me, it looks like the hyphenated version is more frequently used to mean an adjective description of a different noun, like computers, products, problems etc.

File: WindowsServerDocs/identity/ad-ds/plan/Assigning-Domain-Names.md (hunk 2 above)
Comment (illfated) [reply WindowsServerDocs/identity/ad-ds/plan/Assigning-Domain-Names.md:28]: Thanks, that is useful. 👍

File: WindowsServerDocs/identity/ad-ds/plan/Assigning-Domain-Names.md (hunk 1 above)
Comment (illfated) [reply WindowsServerDocs/identity/ad-ds/plan/Assigning-Domain-Names.md:17]: Thank you for the useful input. I will keep this one open for more feedback before committing the change.

File: WindowsServerDocs/identity/ad-ds/plan/Creating-an-Organizational-Unit-Design.md [hunk 3]
```diff
@@ -12,36 +12,33 @@ ms.technology: identity-adds
...
-  
--   [Reviewing OU Design Concepts](../../ad-ds/plan/Reviewing-OU-Design-Concepts.md)  
-  
--   [Delegating Administration by Using OU Objects](../../ad-ds/plan/Delegating-Administration-by-Using-OU-Objects.md)  
-  
+> Applies To: Windows Server 2016, Windows Server 2012 R2, Windows Server 2012
 
+Forest owners are responsible for creating organizational unit (OU) designs for their domains. Creating an OU design involves designing the OU structure, assigning the OU owner role, and creating account and resource OUs.
 
+Initially, design your OU structure to enable delegation of administration. When the OU design is complete, you can create additional OU structures for the application of Group Policy to the users and computers and to limit the visibility of objects. For more information, see Designing a Group Policy Infrastructure ([https://go.microsoft.com/fwlink/?LinkId=106655](https://go.microsoft.com/fwlink/?LinkId=106655)).
```
Comment (JohanFreelancer9) [inline WindowsServerDocs/identity/ad-ds/plan/Creating-an-Organizational-Unit-Design.md:19]:
```suggestion
Initially, design your OU structure to enable delegation of administration. When the OU design is complete, you can create additional OU structures for the application of Group Policy to the users and computers and to limit the visibility of objects. For more information, see [Designing a Group Policy Infrastructure(https://go.microsoft.com/fwlink/?LinkId=106655).
```

File: WindowsServerDocs/identity/ad-ds/plan/Determining-Whether-to-Upgrade-Existing-Domains-or-Deploy-New-Domains.md [hunk 4]
```diff
@@ -12,17 +12,15 @@ ms.technology: identity-adds
...
-  
-For a worksheet to assist you in documenting your plans for new and upgraded domains, download Job_Aids_Designing_and_Deploying_Directory_and_Security_Services.zip from Job Aids for Windows Server 2003 Deployment Kit ([https://go.microsoft.com/fwlink/?LinkID=102558](https://go.microsoft.com/fwlink/?LinkID=102558)) and open "Domain Planning" (DSSLOGI_5.doc).  
-  
+> Applies To: Windows Server 2016, Windows Server 2012 R2, Windows Server 2012
 
+Each domain in your design will either be a new domain or an existing upgraded domain. Users from existing domains that you do not upgrade must be moved into new domains.
+
+Moving accounts between domains can impact end users. Before deciding whether to move users into a new domain or to upgrade existing domains, evaluate the long-term administrative benefits of a new AD DS domain against the cost of moving users into the domain.
+
+For more information about upgrading Active Directory domains to  Windows Server 2008 , see [Upgrading Active Directory Domains to Windows Server 2008 and Windows Server 2008 R2 AD DS Domains](https://technet.microsoft.com/library/cc731188.aspx).
```
Comment (JohanFreelancer9) [inline WindowsServerDocs/identity/ad-ds/plan/Determining-Whether-to-Upgrade-Existing-Domains-or-Deploy-New-Domains.md:21]:
```suggestion
For more information about upgrading Active Directory domains to  Windows Server 2008, see [Upgrading Active Directory Domains to Windows Server 2008 and Windows Server 2008 R2 AD DS Domains](https://technet.microsoft.com/library/cc731188.aspx).
```

File: WindowsServerDocs/identity/ad-ds/plan/Determining-Whether-to-Upgrade-Existing-Domains-or-Deploy-New-Domains.md [hunk 5]
```diff
@@ -12,17 +12,15 @@ ms.technology: identity-adds
...
+
+For more information about restructuring AD DS domains within and between forests, see Active Directory Migration Tool version 3.1 Migration Guide ([https://go.microsoft.com/fwlink/?LinkId=93678](https://go.microsoft.com/fwlink/?LinkId=93678)).
```
Comment (JohanFreelancer9) [inline WindowsServerDocs/identity/ad-ds/plan/Determining-Whether-to-Upgrade-Existing-Domains-or-Deploy-New-Domains.md:23]:
```suggestion
For more information about restructuring AD DS domains within and between forests, see [ADMT Guide: Migrating and Restructuring Active Directory Domains](https://go.microsoft.com/fwlink/?LinkId=93678).
```

File: WindowsServerDocs/identity/ad-ds/plan/Finding-Additional-Resources-for-Logical-Structure-Design.md [hunk 6]
```diff
@@ -11,44 +11,46 @@ ms.technology: identity-adds
...
+
+- For more information about designing the site topology, see [Designing the Site Topology for Windows Server 2008 AD DS](Designing-the-Site-Topology.md).
+
+- For worksheets to assist you in documenting the proposed forest, domain, Domain Name System (DNS) infrastructure, and organizational unit (OU) design, download Job_Aids_Designing_and_Deploying_Directory_and_Security_Services.zip from [Job Aids for Windows Server 2003 Deployment Kit](https://www.microsoft.com/download/details.aspx?id=9608).
+
+- For more information about delegated authentication and constrained delegation, see [Delegating authentication](https://go.microsoft.com/fwlink/?LinkID=106614).
+
+- For more information about configuring firewalls for use with AD DS, see [Active Directory in Networks Segmented by Firewalls](https://go.microsoft.com/fwlink/?LinkId=37928).
+
+- For more information about upgrading Active Directory domains to  Windows Server 2008 , see [Upgrading Active Directory Domains to Windows Server 2008 and Windows Server 2008 R2 AD DS Domains](https://technet.microsoft.com/library/cc731188.aspx).
```
Comment (JohanFreelancer9) [inline WindowsServerDocs/identity/ad-ds/plan/Finding-Additional-Resources-for-Logical-Structure-Design.md:26]:
```suggestion
- For more information about upgrading Active Directory domains to Windows Server 2008, see [Upgrading Active Directory Domains to Windows Server 2008 and Windows Server 2008 R2 AD DS Domains](https://technet.microsoft.com/library/cc731188.aspx).
```

File: WindowsServerDocs/identity/ad-ds/plan/Finding-Additional-Resources-for-Windows-Server-2008-Active-Directory-Site-Topology-Design.md [hunk 7]
```diff
@@ -9,45 +9,43 @@ ms.topic: article
...
-  
--   For more information about how to use the Active Directory Sites and Services snap-in to disable the **Bridge all site links** setting, see Enable or disable site link bridges ([https://go.microsoft.com/fwlink/?LinkId=107073](https://go.microsoft.com/fwlink/?LinkId=107073)).  
-  
--   For information about managing replication through firewalls, see Active Directory in Networks Segmented by Firewalls ([https://go.microsoft.com/fwlink/?LinkId=37928](https://go.microsoft.com/fwlink/?LinkId=37928)).  
-  
--   For more information about read-only domain controller (RODC) features, see AD DS: Read-Only Domain Controllers ([https://go.microsoft.com/fwlink/?LinkID=106616](https://go.microsoft.com/fwlink/?LinkID=106616)).  
-  
--   For information about how to deploy an RODC, see the Step-by-Step Guide for Read-only Domain Controllers ([https://go.microsoft.com/fwlink/?LinkID=92728](https://go.microsoft.com/fwlink/?LinkID=92728)).  
-  
+You can find the following documentation about Active Directory Domain Services (AD DS) on the Windows Server 2003 and  Windows Server 2008  TechCenter Web sites:
```
Comment (JohanFreelancer9) [inline WindowsServerDocs/identity/ad-ds/plan/Finding-Additional-Resources-for-Windows-Server-2008-Active-Directory-Site-Topology-Design.md:16]:
```suggestion
You can find the following documentation about Active Directory Domain Services (AD DS) on the Windows Server 2003 and  Windows Server 2008  TechCenter websites:
```

File: WindowsServerDocs/identity/ad-ds/plan/Finding-Additional-Resources-for-Windows-Server-2008-Active-Directory-Site-Topology-Design.md [hunk 8]
```diff
@@ -9,45 +9,43 @@ ms.topic: article
...
+
+- For more information about designing and deploying print servers, see [Designing and Deploying Print Servers](https://go.microsoft.com/fwlink/?LinkId=107041).
+
+- For more information about spanning trees and Active Directory replication topology, see [Active Directory Replication Topology Technical Reference](https://go.microsoft.com/fwlink/?LinkId=44137).
+
+- For more information about using Adlb.exe and managing environments that have 100 or more branch sites, see [Planning and Deploying Active Directory Domain Services for Branch Offices](https://go.microsoft.com/fwlink/?LinkId=107114).
+
+- For information about installing Network Monitor, see [Monitoring network traffic](https://go.microsoft.com/fwlink/?LinkId=107058).
+
+- For worksheets to assist you in documenting your  Windows Server 2008  AD DS site topology design, see [Job Aids for Windows Server 2003 Deployment Kit](https://www.microsoft.com/download/details.aspx?id=9608).
```
Comment (JohanFreelancer9) [inline WindowsServerDocs/identity/ad-ds/plan/Finding-Additional-Resources-for-Windows-Server-2008-Active-Directory-Site-Topology-Design.md:28]:
```suggestion
- For worksheets to assist you in documenting your  Windows Server 2008 AD DS site topology design, see [Job Aids for Windows Server 2003 Deployment Kit](https://www.microsoft.com/download/details.aspx?id=9608).
```

File: WindowsServerDocs/identity/ad-ds/plan/Determining-Whether-to-Upgrade-Existing-Domains-or-Deploy-New-Domains.md (hunk 4 above)
Comment (illfated) [reply WindowsServerDocs/identity/ad-ds/plan/Determining-Whether-to-Upgrade-Existing-Domains-or-Deploy-New-Domains.md:21]: Thank you, well spotted. 👍

File: WindowsServerDocs/identity/ad-ds/plan/Determining-Whether-to-Upgrade-Existing-Domains-or-Deploy-New-Domains.md (hunk 5 above)
Comment (illfated) [reply WindowsServerDocs/identity/ad-ds/plan/Determining-Whether-to-Upgrade-Existing-Domains-or-Deploy-New-Domains.md:23]: On point. Thank you for the edit.

**Comment by illfated** [comment]
> `[ ... ]` However, "www" has been consistently included throughout the topic in the link to "Job Aids for Windows Server 2003 Deployment Kit", and I would suggest removing it in all the links.

Thanks, I forgot to check that almost all ://microsoft.com/ URLs are compatible with www.microsoft.com -- good point.

**Comment by illfated** [comment]
I will be right back with some updates -- stay safe and well in the meantime.

**Comment by illfated** [comment]
Very well, I think there are just a few more fwlinks which I can easily convert to their actual target page links. Please hold for more updates quite soon™. 😉

**Comment by illfated** [comment]
Also: Thank you @JohanFreelancer9 for your kind assistance so far. Appreciated. 👍

**Comment by illfated** [comment]
There you go. 30 new files, new amount of files: 38. Sorry for all the added trouble of reviewing my changes. In addition to correcting as many links as possible, there are also lots of whitespace consistency adjustments. I can only recommend using the **Hide whitespace changes** feature (https://github.com/MicrosoftDocs/windowsserverdocs/pull/4181/files?diff=unified&w=1) to maintain a modicum of sanity while scrolling through the effective changes, before looking over the whitespace changes I have worked quite a lot on.

**Comment by illfated** [comment]
#sign-off

**Comment by illfated** [comment]
Technical detail correction: I started off with 17 files, so my last commit with 30 files must have contained only 21 new files and 9 old ones.

**Comment by illfated** [comment]
Additional details regarding the files in this PR:
- 37 files from the folder [WindowsServerDocs/identity/ad-ds/plan/](https://github.com/illfated/windowsserverdocs/tree/job-aids-for-ws2k3-deploy-kit/WindowsServerDocs/identity/ad-ds/plan) ((https://github.com/illfated/windowsserverdocs/tree/job-aids-for-ws2k3-deploy-kit/WindowsServerDocs/identity/ad-ds/plan/))
- 1 file from the folder [WindowsServerDocs/identity/ad-ds/plan/security-best-practices/](https://github.com/illfated/windowsserverdocs/tree/job-aids-for-ws2k3-deploy-kit/WindowsServerDocs/identity/ad-ds/plan/security-best-practices) (last file by alphabetical sorting)

**Comment by eross-msft** [comment]
@illfated @JohanFreelancer9  Thank you both for all of your efforts on this content.

**Comment by illfated** [comment]
Very good, thank you as well. I was not expecting to see this PR merged this soon, but then again, it is rather old material and not super consequential.

**Pull request merged**